src/main/java/com/openshift/jenkins/plugins/pipeline/dsl/OpenShiftExec.java

```diff
@@ -0,0 +1,196 @@
+package com.openshift.jenkins.plugins.pipeline.dsl;
+
+import com.openshift.jenkins.plugins.pipeline.Argument;
+import com.openshift.jenkins.plugins.pipeline.ParamVerify;
+import com.openshift.jenkins.plugins.pipeline.model.GlobalConfig;
+import com.openshift.jenkins.plugins.pipeline.model.IOpenShiftExec;
+import hudson.Extension;
+import hudson.model.AbstractBuild;
+import hudson.model.AbstractProject;
+import hudson.model.Action;
+import hudson.model.BuildListener;
+import hudson.tasks.BuildStepMonitor;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepDescriptorImpl;
+import org.jenkinsci.plugins.workflow.steps.Step;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+public class OpenShiftExec extends OpenShiftBaseStep implements IOpenShiftExec {
+
+    protected String pod;
+    protected String container;
+    protected String command;
+    protected List<Argument> arguments = new ArrayList<>();
+    protected String waitTime;
+
+    @DataBoundConstructor
+    public OpenShiftExec( String pod ) {
+        this.pod = pod;
+    }
```

Comment:
In the README you said the command was mandatory, but given the way the ctor is currently laid out, only the pod is mandatory.  At first blush, seems like the command should be mandatory and be part of the ctor.